Update JavaDoc for @DesignRoot.

anssit opened the pull request:
Emphasise that both location and filename for the design file can be specified in the value parameter.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/8805)
<!-- Reviewable:end -->

anssit committed:
Update JavaDoc for @DesignRoot.

Emphasise that both location and filename for the design file can be specified in the value parameter.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vaadin/framework?pullRequest=8805) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vaadin/framework?pullRequest=8805) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vaadin/framework?pullRequest=8805) it.</sub>

hesara commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[server/src/main/java/com/vaadin/annotations/DesignRoot.java, line 32 at r1](https://reviewable.io:443/reviews/vaadin/framework/8805#-KerCNQ7uS8jdtKuLKCw:-KerCNQ84I_vg4gv-oWd:baao3ix) ([raw file](https://github.com/vaadin/framework/blob/0ba02aed572b8556c4449c4111122915e9c2786c/server/src/main/java/com/vaadin/annotations/DesignRoot.java#L32)):*
> ```Java
>  * is loaded from the same package as the annotated class and the design
>  * filename is derived from the class name. You can override the default
>  * behaviour by using the {@link #value()} parameter in order to load the 
> ```

extra whitespace at the end of the line

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/8805)*
<!-- Sent from Reviewable.io -->

anssit committed:
Remove extra whitespace at the end of a line.

anssit commented:
Fixed.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[server/src/main/java/com/vaadin/annotations/DesignRoot.java, line 32 at r1](https://reviewable.io:443/reviews/vaadin/framework/8805#-KerCNQ7uS8jdtKuLKCw:-KerGkn3N9FEUadLEuLh:bss4a2b) ([raw file](https://github.com/vaadin/framework/blob/0ba02aed572b8556c4449c4111122915e9c2786c/server/src/main/java/com/vaadin/annotations/DesignRoot.java#L32)):*
<details><summary><i>Previously, hesara (Henri Sara) wrote…</i></summary><blockquote>

extra whitespace at the end of the line
</blockquote></details>

Removed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/8805#-:-KerGmcpIa0uZ_Vd2dcc:bt2vr1b)*
<!-- Sent from Reviewable.io -->

hesara commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/8805)*
<!-- Sent from Reviewable.io -->
